Library/Homebrew/formula_creator.rb

```diff
@@ -156,10 +156,7 @@ def install
             system "cmake", ".", *std_cmake_args
         <% elsif mode == :autotools %>
             # Remove unrecognized options if warned by configure
```

Comment:
Can we link to https://rubydoc.brew.sh/Formula.html#std_configure_args-instance_method in this comment, so that users know what exactly is going into `std_configure_args`? Especially given that the comment might be asking them to remove options they don't even know are there...

Reply:
Should links be added for the other `std_*_args` also? Why add this link when there is already the link to https://rubydoc.brew.sh/Formula above?

Reply:
>Should links be added for the other `std_*_args` also?

Perhaps, but that's less important than for `std_configure_args`. There's very little need to know what the other `std_*_args` say: you can just include them and they'll just work. Not sure we can say the same for `std_configure_args`.

>Why add this link when there is already the link to https://rubydoc.brew.sh/Formula above?

Same reasoning as above.

Reply:
Agreed with @carlocab